src/python/module/smartredis/util.py

```diff
@@ -24,16 +24,18 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from .error import *
+import typing as t
 from functools import wraps
+
+import numpy as np
+
```

Comment:
I don't understand how we got to this ordering

Reply:
me neither. i didn't really inspect the result of isort but I'm glad to see if another preferred ordering doesn't trigger a change

Reply:
Busted out [isort docs](https://pycqa.github.io/isort/docs/configuration/custom_sections_and_ordering.html) and it does make sense, now. 

Default sort ordering is: `FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER`

numpy is moved out of the stdlib section into a third-party grouping, per that default.